fedimint-server/src/net/p2p_connector.rs

```diff
@@ -358,7 +355,7 @@ where
             .node_ids
             .iter()
             .find(|entry| entry.1 == &node_id)
-            .context("Node id {node_id} is unknown")?
+            .context(format!("Node id {node_id} is unknown"))?
```

Comment:
`with_context`? Otherwise it allocates all the time?

Reply:
Overlooked, fixing now